openedx/core/djangoapps/content_libraries/tests/test_content_libraries.py

```diff
@@ -454,8 +454,6 @@ def test_library_blocks_limit(self):
             lib_id = lib["id"]
             block_data = self._add_block_to_library(lib_id, "unit", "unit1")
             # Second block should throw error
-            with self.assertRaises(BlockLimitReachedError):
-                self._add_block_to_library(lib_id, "problem", "problem1")
+            self._add_block_to_library(lib_id, "problem", "problem1", expect_response=400)
             # Also check that limit applies to child blocks too
-            with self.assertRaises(BlockLimitReachedError):
-                self._add_block_to_library(lib_id, "html", "html1", parent_block=block_data["id"])
+            self._add_block_to_library(lib_id, "html", "html1", expect_response=400)
```

Comment:
Oh @mavidser I just noticed an issue here: the comment says "Also check that limit applies to child blocks too" but then after you refactored this it's missing the `parent_block=block_data["id"]` part, so it's not checking a child block anymore. Please add that back in then squash and I'll approve and merge.

Reply:
@mavidser ping ^ :)

Reply:
Oops, my bad. Have fixed! :)